xchange-ftx/src/main/java/org/knowm/xchange/ftx/FtxAuthenticated.java

```diff
@@ -194,13 +194,13 @@ FtxResponse changeSubAccountName(
 
   @DELETE
   @Path("/orders/by_client_id/{client_order_id}")
-  FtxResponse<FtxOrderDto> cancelOrderByClientId(
-      @HeaderParam("FTX-KEY") String apiKey,
-      @HeaderParam("FTX-TS") Long nonce,
-      @HeaderParam("FTX-SIGN") ParamsDigest signature,
-      @HeaderParam("FTX-SUBACCOUNT") String subaccount,
-      @PathParam("client_order_id") String clientOrderId)
-      throws IOException, FtxException;
+  FtxResponse<String> cancelOrderByClientId(
```

Comment:
why are you moving to a string?

Reply:
Because the response is a string and i got error when i had something else.

Reply:
What was the error message